live-ingester/src/main/java/org/alfresco/hxi_connector/live_ingester/adapters/config/messaging/LiveIngesterMessagingConfig.java

```diff
@@ -66,7 +70,11 @@ public DiscoveryApiClient discoveryApiClient(IntegrationProperties integrationPr
     @Bean
     public ApplicationInfoProvider applicationInfoProvider(DiscoveryApiClient discoveryApiClient, IntegrationProperties integrationProperties)
     {
-        return new ApplicationInfoProvider(discoveryApiClient, integrationProperties.application(), integrationProperties.alfresco().repository().versionOverride());
+        return new ApplicationInfoProvider(discoveryApiClient, integrationProperties.application(), Optional.of(integrationProperties)
+                .map(IntegrationProperties::alfresco)
+                .map(Alfresco::repository)
+                .map(Repository::versionOverride)
+                .filter(Predicate.not(String::isBlank)));
```

Comment:
Please let's format this in a way that's more readable if possible, e.g. something like:

```java
return new ApplicationInfoProvider(
        discoveryApiClient,
        integrationProperties.application(),
        Optional.of(...)
    );
```

Os simply extract the `Optional.of(...)` chain to a local variable, and then keep the constructor invocation as a one-liner.

Reply:
Done.